packages/orchestration/src/utils/start-helper.js

```diff
@@ -43,6 +43,7 @@ import { makeZoeTools } from './zoe-tools.js';
  * @param {Baggage} baggage
  * @param {OrchestrationPowers} remotePowers
  * @param {Marshaller} marshaller
+ * @internal
```

Comment:
doesn't seem to have the desired effect. I still see it:

![image](https://github.com/user-attachments/assets/2235aa81-e777-431b-a47d-321c4ce23061)


https://ta-export-withorchestration.agoric-sdk.pages.dev/modules/_agoric_orchestration

Reply:
I tried merging and that didn't fix it. Now I suspect a bug in typedoc

Reply:
yeah, I've tried without luck to get `@internal` to do anything